commands.txt

```diff
@@ -1,3 +1,3 @@
 cd $SD_ARTIFACTS_DIR
 find . -type f -print > manifest.txt
-find . -type f -exec sh -c 'f=`echo $1 | sed "s/^\.\///"`; curl -H "Authorization: Bearer $SD_TOKEN" -H "Content-Type: text/plain" -X PUT $STORE_URL/v1/builds/$SD_BUILD_ID/$ARTIFACTS_DIR_SUFFIX/$f -T ./$f' -- {} \;
+find . -type f -exec sh -c 'f=`echo $1 | sed "s/^\.\///"`; encoded=`curl -s -w "%{url_effective}\n" --data-urlencode $f -G "" | sed -e "s/^\/?//"`; curl -H "Authorization: Bearer $SD_TOKEN" -H "Content-Type: text/plain" -X PUT $STORE_URL/v1/builds/$SD_BUILD_ID/$ARTIFACTS_DIR_SUFFIX/$encoded -T ./$f' -- {} \;
```

Comment:
I think some image including old `curl` (before "7.18.0") cannot use `--data-urlencode` option.
https://curl.haxx.se/docs/manpage.html

Reply:
Oh, I see.
So it's better to use sd-step in order to use latest version.